system/Database/ResultInterface.php

```diff
@@ -16,14 +44,15 @@
  */
 interface ResultInterface
 {
+
 	/**
 	 * Retrieve the results of the query. Typically an array of
 	 * individual data rows, which can be either an 'array', an
 	 * 'object', or a custom class name.
 	 *
 	 * @param string $type The row type. Either 'array', 'object', or a class name to use
 	 *
-	 * @return array
+	 * @return mixed
```

Comment:
Why changed to `mixed`?

Reply:
As I mentioned elsewhere, this first attempt at a PR had outdated files as its starting point. That is why I closed this request and tried again. Thank you for look at this PR, but I want you to know that I've closed it and it should be ignored. I'm sorry for wasting your time :(